charts/opssight/values.yaml

```diff
@@ -0,0 +1,139 @@
+# Default values for Opssight.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+# Docker registry to pull Black Duck images
+registry: docker.io/blackducksoftware
+imageTag: 2.2.5
+
+# image pull secret to download the Black Duck images (mostly applicable for air gapped customers)
+imagePullSecrets: [] # array of strings delimited by comma
+
+# This is used to start or stop the app. Set to Running to start, or Stopped to stop.
+status: Running
+
+# Secrets base64 encoded
+secret:
+  externalHub: e30=
+  securedRegistries: e30=
+
+# Specifications as it will go to configMap
+spec:
```

Comment:
why it's having spec?

Reply:
remove spec and move all the metrics related parameters under prometheus pod

Reply:
Should I just remove Spec?